bin/main.rb

```diff
@@ -0,0 +1 @@
+puts 'Hello World!'
```

Comment:
A line with commented `#!/usr/bin/env ruby` is required here at the top of the main.rb file.